cmd/geth/main.go

```diff
@@ -175,6 +175,9 @@ var (
 		utils.BuilderBeaconEndpoints,
 		utils.BuilderRemoteRelayEndpoint,
 		utils.BuilderSecondaryRemoteRelayEndpoints,
+		utils.BuilderRateLimitDuration,
```

Comment:
Would be nice to also update the readme flags, usually best to just run `geth --help` and copy the relevant new flags

Reply:
ohh i thought you meant just the env var, my bad, updated!